src/components/Editor/CodeEditor/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
I think disabling the lint rules makes it more explicit to specify line numbers using `eslint-disable-line` or `eslint-disable-next-line` comment.

> Reference: [Disabling Rules - ESLint](https://eslint.org/docs/latest/user-guide/configuring/rules#disabling-rules)

@gollumnima @blurfx Would it be okay could I work on this proposal? 🥺

Reply:
@x86chi Yes, and if you found an unintentional empty dependency array, could you please update it instead of using eslint-disable comment?

Reply:
Sure!